docs/time/performance.inc.rst

```diff
@@ -10,3 +10,49 @@ Performance Tips
 
 Here we provide some tips and tricks for how to optimize performance of code
 using `astropy.time`.
+
+Listed below are two approaches to calculating light travel times for tens of
+thousands of sources in a degree patch of the sky.
+
+>>> import numpy as np
+>>> import astropy.coordinates as coord
+>>> import astropy.units as u
+>>> from astropy.time import Time
+
+>>> ra = np.random.normal(0.0, 1.0, 50000)
+>>> dec = np.random.normal(0.0, 1.0, 50000)
+>>> coos = coord.SkyCoord(ra, dec, unit=u.deg)
+
+   Lapalma
+
+>>> observatory = coord.EarthLocation.from_geocentric(5327448.9957829, -1718665.73869569, 3051566.90295403, unit='m')
+
+*The first approach and its time to completion below:*
```

Comment:
Why is this in italics?  Maybe just regular text is sufficient?  Or it could be a sub-sub-heading:
```
First approach
^^^^^^^^^^^^^^
```

Reply:
I will turn it into a sub sub heading